BUILD

```diff
@@ -6,16 +6,16 @@ container_bundle(
     name = "all",
     images = {
         "gcr.io/{PROJECT_ID}/base:debug": "//base:debug",
-        "gcr.io/{PROJECT_ID}/base:debug-debian10": "//base:debug-debian10",
+        "gcr.io/{PROJECT_ID}/base:debug_debian10": "//base:debug_debian10",
```

Comment:
The names in the front that starts with gcr.io are Docker image references, so tags should continue to use hyphens. But the names in the end are Bazel targets, so it is correct to use underscores.

Reply:
Nice catch thank you will fix!